docs/_docs/General/best-practices.md

```diff
@@ -7,12 +7,21 @@ toc: false
 
 This page describes best practices for writing your scripts.
 
-1. Fork the repository
-2. Create a new branch for your script
+When writing your own script you'll need to test it. To test a script Phoenicis doesn't only require the script, but also some context information. For example a miniature image used in the library after the script has been installed. Some other context information is extracted by Phoenicis via the directory structure, the script is located in.
+
+Therefore to test a script it needs to be embedded inside a repository. For this a local repository, which is based on the official script repository, is recommended. To setup your own local repository proceed the following steps:
+
+1. Fork the official scripts repository
```

Comment:
We should change the README to use a fork as well.

Reply:
Yes, you're right